### ISSUE TITLE
The trending ports does not have 'isSubscribed' value to know if the user is subscribed to it

### ISSUE THREAD

**Issue by derickfan** [DataMascara/cisc3140-su19-project#28]
In the portIndex.html file it checks if the port['isSubscribed] is True so it'll know whether to show "Subscribe" or "Joined"
![Screen Shot 2019-08-11 at 7 50 49 PM](https://user-images.githubusercontent.com/22162773/62841041-55e66680-bc71-11e9-8503-aff128488e41.png)

However in the base.html file the trending ports does not check for the 'isSubscribed' value so it will always show the user "Subscribe" even if they are already subscribed to it
![Screen Shot 2019-08-11 at 7 45 17 PM](https://user-images.githubusercontent.com/22162773/62841053-875f3200-bc71-11e9-86b2-f640d630d79f.png)

![Screen Shot 2019-08-11 at 7 52 41 PM](https://user-images.githubusercontent.com/22162773/62841055-99d96b80-bc71-11e9-832f-9915dfe090da.png)

**Comment by Maker-Mark** [comment]
> @vladislavglad , great thanks for the re-deployment! It is extremely helpful for visualization.
Live deployed app is at https://bc-app-class.herokuapp.com/

**Comment by mary060196** [comment]
Good morning, @derickfan and @Maker-Mark !
The trending ports issues with the subscription buttons were fixed.
Now:
- The trending ports array of dicts can include an `isSubscribed` argument that will be caught by the template `base.html`. What it means: a user can see to which of the trending ports he or she is already subscribed.
- The `scripts.js` file was modified to check for existence of additional buttons with regard to the same port. What it means: if a user presses Subscribe or Joined (to unsubscribe) on the trending ports section, and the user is at the page that renders `portIndex.html`, the corresponding button on the port index will change accordingly.
- The `scriptsPortIndex.js` file was modified to check for the existence of a button on the trending port section for the same port. What it means: if a user presses Subscribe or Joined (to unsubscribe) on the port index section, and the user is at the page that renders `portIndex.html`, the corresponding button on the trending ports section will change accordingly.
- Since the `Subscriptions` tab in `Dashboard` also displays ports, the exact same features above were added for the `scriptsUserInfo.js` file.
In other words, please check if this particular bug was fixed.

Thank you very, very much for your patience!

**Comment by mary060196** [comment]
@derickfan ,
I have made changes to two JavaScript files to correct the issue with the buttons, but re-deployment did not occur. I am using the GitHub GUI, so I do not push with the command line. How can the app be re-deployed?
Thank you for this!

**Comment by mary060196** [comment]
Issue successfully resolved! Thanks a lot!
